Source/Core/DolphinQt/Config/ControllersWindow.cpp

```diff
@@ -199,15 +200,17 @@ void ControllersWindow::CreateWiimoteLayout()
   m_wiimote_layout->addWidget(m_wiimote_speaker_data, m_wiimote_layout->rowCount(), 1, 1, -1);
 }
 
-void ControllersWindow::CreateAdvancedLayout()
+void ControllersWindow::CreateCommonLayout()
 {
-  m_advanced_box = new QGroupBox(tr("Advanced"));
-  m_advanced_layout = new QHBoxLayout();
-  m_advanced_bg_input = new QCheckBox(tr("Background Input"));
+  m_common_box = new QGroupBox(tr("Common"));
```

Comment:
Could you add an `i18n` comment above this line so that translators are less likely to mistranslate this? Something like:

`// i18n: This is "common" as in "shared", not the opposite of "uncommon"`